converter/src/test/java/gov/cms/qpp/conversion/validate/ClinicalDocumentValidatorTest.java

```diff
@@ -255,19 +255,6 @@ void testInvalidProgramName() {
 				.containsExactly(ProblemCode.CLINICAL_DOCUMENT_INCORRECT_PROGRAM_NAME.format(invalidProgramName, ClinicalDocumentValidator.VALID_PROGRAM_NAMES));
 	}
 
-	@Test
-	void testMissingVirtualGroupId() {
-		Node clinicalDocumentNode = createValidClinicalDocumentNode();
-		clinicalDocumentNode.putValue(ClinicalDocumentDecoder.ENTITY_TYPE, ClinicalDocumentDecoder.ENTITY_VIRTUAL_GROUP);
-		Node aciSectionNode = createAciSectionNode(clinicalDocumentNode);
-		clinicalDocumentNode.addChildNode(aciSectionNode);
-		ClinicalDocumentValidator validator = new ClinicalDocumentValidator();
-		List<Detail> errors = validator.validateSingleNode(clinicalDocumentNode).getErrors();
-
-		assertThat(errors).comparingElementsUsing(DetailsErrorEquals.INSTANCE)
-			.containsExactly(ProblemCode.VIRTUAL_GROUP_ID_REQUIRED);
-	}
-
```

Comment:
Could you add this test back if the file, since we added back the virtual group logic?